src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    public static final String EMPTY_FILE = "src/test/resources/empty_test.csv";
+    public static final String WRONG_PATH = "src/test/test_file.txt";
+    public static final String INPUT_FILE_PATH = "src/test/resources/input_test.csv";
+    public static final String HEADER = "operation,fruit,amount";
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void beforeAll() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void readFromFile_Ok() {
```

Comment:
```suggestion
    public void readFromFile_ok() {
```
each part of naming should start with lower case, the same for other test methods